.github/workflows/ci.yml

```diff
@@ -25,36 +25,32 @@ jobs:
       with:
         fetch-depth: 0  # unshallow fetch for setuptools-scm
 
-    - name: Use MSBuild (Windows)
-      uses: microsoft/setup-msbuild@v1.0.2
-      if: matrix.os == 'windows-latest'
-
-    - uses: actions/setup-python@v1
-      name: Install Python
-      with:
-        python-version: '3.6'
-
-    - name: Install cibuildwheel
-      run: |
-        python -m pip install cibuildwheel==1.5.2
-
-    - name: Build wheel
-      run: |
-        python -m cibuildwheel --output-dir dist
+    - name: Build wheels
+      uses: pypa/cibuildwheel@v2.2.2
       env:
         # The packaged FreeType library is independent of the Python ABI so we only
         # need to build it once.
-        CIBW_BUILD: "cp36-*x86_64 cp36-*amd64"
+        CIBW_BUILD: "cp39-*"
+        CIBW_ARCHS: "auto64"
+        CIBW_ARCHS_MACOS: "auto64 universal2 arm64"
         CIBW_ENVIRONMENT: "FREETYPEPY_BUNDLE_FT=yes PYTHON_ARCH=64"
         CIBW_MANYLINUX_X86_64_IMAGE: manylinux1
         CIBW_TEST_COMMAND: "pytest {project}/tests"
         CIBW_TEST_REQUIRES: "pytest"
+      with:
+        output-dir: dist
 
     - uses: actions/upload-artifact@v2
       with:
         name: freetype-py-dist
         path: dist/*.whl
 
+    - uses: actions/setup-python@v1
+      name: Install Python
+      if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')

```

Comment:
the whole building of sdist and publishing of wheels could be moved to a separate "deploy" job that is conditioned on the "build" job passing and is only triggered on tagged commits (so you don't have to repeat the `if:` condition every time).

Reply:
Sure, I also prefer that but I didn't want to refactor your CI pipeline without consent :) I guess this is the green light to do that then.

Reply:
Done.